ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg-nonprod.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
```

Comment:
There is already existing file "AuthorisationCaseEvent-SDO-R2-nonprod.json" move your changes to that file. FYI: Our all changes should be in nonprod files and its name should contain "SDO-R2".